src/php/bin/xds_manager.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+# Copyright 2021 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Manage PHP child processes for the main PHP xDS Interop client"""
+
+import argparse
+import fcntl
+import os
+import subprocess
+
+# This script is being launched from src/php/bin/run_xds_client.sh
+# to manage PHP child processes which will send 1 RPC each
+# asynchronously. This script keeps track of all those open
+# processes and reports back to the main PHP interop client each
+# of the child RPCs' status code.
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--tmp_file1', nargs='?', default='')
+    parser.add_argument('--tmp_file2', nargs='?', default='')
+    parser.add_argument('--bootstrap_path', nargs='?', default='')
+    args = parser.parse_args()
+    server_address = ''
+    rpcs_started = []
+    open_processes = {}
+    client_env = dict(os.environ)
+    client_env['GRPC_XDS_BOOTSTRAP'] = args.bootstrap_path
+    client_env['GRPC_XDS_EXPERIMENTAL_V3_SUPPORT'] = 'true'
+    client_env['GRPC_XDS_EXPERIMENTAL_ENABLE_TIMEOUT'] = 'true'
```

Comment:
Why are these set explicitly, instead of picking them up from the environment?

Reply:
Comment addressed. Removed the hardcoded env var.